README.md

```diff
@@ -5,58 +5,73 @@
+
+### Nimble packages
+
+The general-use Nimble packages implementing the Nimble Design System.
+
 [![Nimble Angular NPM version and repo link](https://img.shields.io/npm/v/@ni/nimble-angular.svg?label=@ni/nimble-angular)](https://www.npmjs.com/package/@ni/nimble-angular)
 [![Nimble Blazor Nuget version and repo link](https://img.shields.io/nuget/v/NimbleBlazor.svg?label=NimbleBlazor)](https://www.nuget.org/packages/NimbleBlazor)
 [![Nimble Components NPM version and repo link](https://img.shields.io/npm/v/@ni/nimble-components.svg?label=@ni/nimble-components)](https://www.npmjs.com/package/@ni/nimble-components)
-[![Nimble Tokens NPM version and repo link](https://img.shields.io/npm/v/@ni/nimble-tokens.svg?label=@ni/nimble-tokens)](https://www.npmjs.com/package/@ni/nimble-tokens)
 
-The NI Nimble Design System: styled UI components for NI applications.
+- **[`@ni/nimble-angular`](/angular-workspace/projects/ni/nimble-angular/)** - Handy bindings to the Nimble components for Angular applications.
+- **[`@ni/nimble-blazor`](/packages/blazor-workspace/)** - Handy bindings to the Nimble components for Blazor applications.
+- **[`@ni/nimble-components`](/packages/nimble-components/)** - Nimble components to be used by [any type of  application](https://custom-elements-everywhere.com/).
 
-[![storybook page](https://img.shields.io/badge/storybook-white.svg?logo=storybook)](https://ni.github.io/nimble/storybook)
-[![example angular app](https://img.shields.io/badge/example%20angular%20app-dd0031.svg?logo=angular)](https://ni.github.io/nimble/storybook/example-client-app)
-[![example blazor app](https://img.shields.io/badge/example%20blazor%20app-572b8a.svg?logo=blazor)](https://ni.github.io/nimble/storybook/blazor-client-app/wwwroot)
+### Spright packages
 
-If you are at NI, lucky you! **Reach out to ask questions directly in the [Design System Teams channel](https://teams.microsoft.com/l/team/19%3awo8vmMKMsHfltKXxc0bczZo-X4JlQSV5VxpaRJdh13k1%40thread.tacv2/conversations?groupId=9ee065d7-3898-4245-82f6-76e86084b8b1&tenantId=87ba1f9a-44cd-43a6-b008-6fdb45a5204e).**
+The context-specific Spright packages used for specific applications.
 
-## Getting Started
+[![Spright Angular NPM version and repo link](https://img.shields.io/npm/v/@ni/spright-angular.svg?label=@ni/spright-angular)](https://www.npmjs.com/package/@ni/spright-angular)
+[![Spright Blazor Nuget version and repo link](https://img.shields.io/nuget/v/SprightBlazor.svg?label=SprightBlazor)](https://www.nuget.org/packages/SprightBlazor)
+[![Spright Components NPM version and repo link](https://img.shields.io/npm/v/@ni/spright-components.svg?label=@ni/spright-components)](https://www.npmjs.com/package/@ni/spright-components)
 
-This repository contains the source for the following packages:
+- [`@ni/spright-angular`](/angular-workspace/projects/ni/spright-angular/) - Handy bindings to the Spright components for Angular applications.
```

Comment:
nit: The tone is friendly, but 'handy' might be too casual - it also makes me think of https://www.handy.com

```suggestion
- [`@ni/spright-angular`](/angular-workspace/projects/ni/spright-angular/) - Angular bindings for Spright components.
```

Reply:
I was trying to convey a tone of "these are handy / useful but technically not strictly necessary".
I can go with the straightforward message if preferred. Would the less colloquial "useful" be a good alternate?

```suggestion
- [`@ni/spright-angular`](/angular-workspace/projects/ni/spright-angular/) - Useful bindings to the Spright components for Angular applications.
```